src/main/kotlin/no/nav/klage/oppgave/repositories/SaksbehandlerRepository.kt

```diff
@@ -112,7 +112,7 @@ class SaksbehandlerRepository(
 
     private fun List<String>.hasRole(role: String) = any { it.contains(role) }
 
-    fun getNameForSaksbehandler(navIdent: String): String {
+    fun getSammensattNavnForSaksbehandler(navIdent: String): String {
         return azureGateway.getPersonligDataOmSaksbehandlerMedIdent(navIdent).sammensattNavn
```

Comment:
Finnes det flere egenskaper for navn som kommer fra Azure, eller heter den bare `sammensattNavn` uten at det finnes egenskaper for ikke-sammensatt navn?

Reply:
Jeg ser vi får ut denne lista fra Azure, og så mapper vi den videre: 
```
return SaksbehandlerPersonligInfo(
            data.onPremisesSamAccountName,
            data.id,
            data.givenName,
            data.surname,
            data.displayName,
            data.mail
        )
 ```
 Har dere behov for noe annet enn `sammensattNavn` her?

Reply:
Behov? Nei.
Hadde det vært fint om alle navn var strukturert likt? Ja.

Azure gir da både fornavn og etternavn, men ikke mellomnavn. Vi kunne vurdert å la mellomnavn være tomt og bruke fornavn og etternavn som ellers.